react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+export function ToggleSwitch() {
+  const [onOrOff, setOnOrOff] = useState('toggle-switch is-on');
```

Comment:
You can use this text as a value like you've done, but it means that if you change your class names you'll have to change your code in several places, as well. It would be simpler to just make this a boolean and then compute the class name based on that boolean value.